apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -0,0 +1,650 @@
+
+
+describe('B2BAppAccessRightSet', () => {
+    let admin
+    let support
+    let integratedServiceUser, integratedToAnotherAppServiceUser
+    let notServiceUser
+    let anonymous
+
+    let app
+    let anotherApp
+
+    let accessRightSet
+
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient()
+        support = await makeClientWithSupportUser()
+        notServiceUser = await makeClientWithNewRegisteredAndLoggedInUser()
+        anonymous = await makeClient()
+
+        const [createdApp] = await createTestB2BApp(admin)
+        app = createdApp
+
+        const [createdApp2] = await createTestB2BApp(admin)
+        anotherApp = createdApp2
+
+        const [createdAccessRightSet] = await createTestB2BAppAccessRightSet(admin, app)
+        accessRightSet = createdAccessRightSet
+
+        const [newServiceUser] = await registerNewServiceUserByTestClient(admin)
+        integratedServiceUser = await makeLoggedInClient({ email: newServiceUser.email, password: newServiceUser.password })
+        await createTestB2BAppAccessRight(admin, integratedServiceUser.user, app)
+
+        const [newServiceUser2] = await registerNewServiceUserByTestClient(admin)
+        integratedToAnotherAppServiceUser = await makeLoggedInClient({ email: newServiceUser2.email, password: newServiceUser2.password })
+        await createTestB2BAppAccessRight(admin, integratedToAnotherAppServiceUser.user, anotherApp)
+    })
+
+    describe('Accesses', () => {
+        describe('Admin', () => {
+            test('Can create', async () => {
+                const [app] = await createTestB2BApp(admin)
+                const [createdAccessRightSet] = await createTestB2BAppAccessRightSet(admin, app)
+
+                expect(createdAccessRightSet).toBeDefined()
+                expect(createdAccessRightSet.id).toMatch(UUID_RE)
+                expect(createdAccessRightSet).toHaveProperty('app.id', app.id)
+                expect(createdAccessRightSet).toHaveProperty('canManageOrganizations', false)
+                expect(createdAccessRightSet).toHaveProperty('canReadOrganizations', false)
+                expect(createdAccessRightSet).toHaveProperty('canManageContacts', false)
+                expect(createdAccessRightSet).toHaveProperty('canReadContacts', false)
+                expect(createdAccessRightSet).toHaveProperty('canManageProperties', false)
+                expect(createdAccessRightSet).toHaveProperty('canReadProperties', false)
+            })
+
+            test('Can read everything', async () => {
+                const [foundedAccessRightSet] = await B2BAppAccessRightSet.getAll(admin, {
+                    id: accessRightSet.id,
+                })
+                expect(foundedAccessRightSet).toHaveProperty('id', accessRightSet.id)
+            })
+
+            test('Can update', async () => {
+                const [updatedAccessRightSet] = await updateTestB2BAppAccessRightSet(admin, accessRightSet.id)
```

Comment:
Why empty payload? :(

Just to check its possible to update (not representative for real-life case imo)